singleheader/amalgamate.sh

```diff
@@ -127,8 +127,17 @@ echo "/* auto-generated on ${timestamp}. Do not edit! */" > ${AMAL_C}
     done
 } >> ${AMAL_C}
 
-cp -f "${SCRIPTPATH}/amalgamate_demo.cpp" "${DEMOCPP}"
-cp -f "${SCRIPTPATH}/README.md" "${README}"
+function docopy()
+{
+  if [ "$1" -ef "$2" ]; then
+    echo "I would copy $1 to $2 but they are the same file.";
```

Comment:
there is no need to log that message. All we really want to do is to copy these files if output dir isn't the same as singleheader dir. See udpated PR: https://github.com/simdjson/simdjson/pull/1113/files